cockroachdb/values.yaml

```diff
@@ -384,6 +409,17 @@ tls:
       kind: Issuer
       name: cockroachdb
 
+  generator:
+    # Image to use for generating certificates
+    image:
+      repository: cockroachdb/cockroach-generate-cert
```

Comment:
does this image actually exist or is this a placeholder?

Reply:
This is a placeholder. We can decide on the image name for this. This image will be build once we have the CI workflows in place.

Reply:
can we make it cockroachdb/cockroach-generate-cert-placeholder and change line 413 to reflect that this needs to be updated then?

Reply:
Done